modules/pipeline/providers/leaderworker/worker/error.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package worker
+
+import (
+	"fmt"
+)
+
+var (
+	ErrNotFound          = fmt.Errorf("not found")
+	ErrNilWorker         = fmt.Errorf("nil worker")
+	ErrNoWorkerID        = fmt.Errorf("no worker id")
+	ErrNoWorkerType      = fmt.Errorf("no worker type")
+	ErrInvalidWorkerType = fmt.Errorf("invalid worker type")
+	ErrNoCreatedTime     = fmt.Errorf("no created time")
+	ErrMarshal           = fmt.Errorf("cannot marshal")
```

Comment:
how to do i18n?

Reply:
no need at pipeline level, it's only internal use.